common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,668 @@
+darkspear_trollish = {
+	template = "night_elven"
+	
+	skin_color = {
+		10 = { 0.78 0.35 0.78 0.35 } 
+		10 = { 0.72 0.34 0.72 0.34 }
+		10 = { 0.70 0.37 0.70 0.37 } 
+		10 = { 0.74 0.40 0.74 0.40 }
+		10 = { 0.678 0.447 0.678 0.447 }
+		10 = { 0.608 0.333 0.608 0.333 }
+		10 = { 0.502 0.341 0.502 0.341 }
+		10 = { 0.733 0.333 0.733 0.333 }
+		10 = { 0.694 0.388 0.694 0.388 }
+		10 = { 0.620 0.408 0.620 0.408 }
+		10 = { 0.737 0.427 0.737 0.427 }
+		10 = { 0.784 0.365 0.784 0.365 }
+	}
+	eye_color = {
+        10 = { 1 0.58 1 0.58 }
+		10 = { 1 0.51 1 0.51 }
+		10 = { 0.082 0.608 0.082 0.608 }
+		10 = { 0.176 0.608 0.176 0.608 }
+		10 = { 0.569 0.541 0.569 0.541 }
+		10 = { 0.408 0.588 0.408 0.588 }
+		10 = { 0.714 0.541 0.714 0.541 }
+		10 = { 0.718 0.718 0.718 0.718 }
+		10 = { 0.988 0.643 0.988 0.643 }
+		10 = { 0.988 0.573 0.988 0.573 }
+		10 = { 0.894 0.576 0.894 0.576 }
+	}
+	hair_color = {
+		10 = { 0 0.73 0 0.73 }
+		10 = { 0.95 0.6 0.95 0.6 }
+		10 = { 0.04 0.58 0.04 0.58 }
+		10 = { 0.11 0.58 0.11 0.58 }
+		10 = { 0.4 0.79 0.4 0.79 }
+		10 = { 0.53 0.82 0.53 0.82 }
+		10 = { 0.68 0.77 0.68 0.77 }
+		10 = { 0.77 0.87 0.77 0.87 }
+		10 = { 0.77 0.87 0.77 0.87 }
+		10 = { 0.69 0.30 0.69 0.30 }
+		10 = { 0.69 0.14 0.69 0.14 }
+		10 = { 0.84 0.77 0.84 0.77 }
+		10 = { 0.95 0.33 0.95 0.33 }
+	}
```

Comment:
Remember, you can use ranges here.

Reply:
I used ranges for the other ethnicities. I should probably go back and add some variety to these as well.

Reply:
Updated these in 491518f3192e0a590c69c9441dfedfa1526c353e.